backend/api/v1/dependencies.py

```
@@ -0,0 +1,112 @@
+from ...infrastructure.zfs.repositories.zfs_pool_repository_impl import ZFSPoolRepositoryImpl
+
+from ...application.zfs.dataset_management_service import DatasetManagementService
+from ...application.zfs.snapshot_management_service import SnapshotManagementService
+from ...application.zfs.pool_management_service import PoolManagementService
+from ...application.docker.docker_management_service import DockerManagementService
+from ...application.migration.migration_orchestration_service import MigrationOrchestrationService
+
+# For now, we'll create mock implementations for Docker repositories
+# In production, these would be real implementations
+from ...core.interfaces.docker_repository import (
+    DockerContainerRepository, DockerImageRepository, DockerNetworkRepository,
```

Ruff (0.11.9)

`...core.interfaces.docker_repository.DockerContainerRepository` imported but unused

Remove unused import

(F401)